src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE_PER_DAY = 40;
+  const BASIC_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const basePrice = days * BASE_PRICE_PER_DAY;
+
+  let totalDiscount = 0;
+
+  if (days >= LONG_TERM) {
+    totalDiscount = LONG_TERM_DISCOUNT;
+  } else if (days >= 3) {
+    totalDiscount = BASIC_DISCOUNT;
+  }
```

Comment:
The logic here only applies one discount, either the long-term or the basic discount. According to the task description, both discounts should be applied when applicable. Consider modifying the logic to add both discounts together when the conditions for both are met.